tracer/src/Datadog.Trace/Telemetry/Transports/JsonTelemetryTransport.cs

```diff
@@ -52,29 +56,35 @@ public async Task<TelemetryPushResult> PushTelemetry(TelemetryData data)
                 Log.Debug("Telemetry sent successfully");
                 return TelemetryPushResult.Success;
             }
-            else if (response.StatusCode == 404)
+
+            if (HandleErrorResponse(response) is { } result)
             {
-                Log.Debug("Error sending telemetry: 404. Disabling further telemetry, as endpoint '{Endpoint}' not found", _requestFactory.Info(_endpoint));
-                return TelemetryPushResult.FatalError;
+                return result;
             }
-            else
+
+            if (response.StatusCode == 404)
             {
-                Log.Debug<string, int>("Error sending telemetry to '{Endpoint}' {StatusCode} ", _requestFactory.Info(_endpoint), response.StatusCode);
-                return TelemetryPushResult.TransientFailure;
+                Log.Debug("Error sending telemetry: 404. Disabling further telemetry, as endpoint '{Endpoint}' not found", GetEndpointInfo());
+                return TelemetryPushResult.FatalError;
             }
+
+            Log.Debug<string, int>("Error sending telemetry to '{Endpoint}' {StatusCode} ", GetEndpointInfo(), response.StatusCode);
+            return TelemetryPushResult.TransientFailure;
         }
         catch (Exception ex) when (IsFatalException(ex))
         {
-            Log.Warning(ex, "Error sending telemetry data, unable to communicate with '{Endpoint}'", _requestFactory.Info(_endpoint));
+            Log.Information(ex, "Error sending telemetry data, unable to communicate with '{Endpoint}'", GetEndpointInfo());
```

Comment:
What was the reason for changing this from `Warning` level to `Information` level? Warning seemed like an appropriate level to me

Reply:
> What was the reason for changing this from Warning level to Information level

Some customers take the warnings seriously, and raise support requests about them 😅 For telemetry, we _expect_ that we might not be able to send telemetry data (e.g. their agent is too old, we're running in an environment that _hasn't_ been configured etc), so the warning isn't always valid. 

Information messages are still written by default, so it seemed sufficient to me?

Reply:
Makes sense, thanks!